_posts/2023-03-27-exploring-a-world-of open-source.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Exploring a world of open source"
+date: 2023-03-27
+author: Jan Ainali
+type: blogpost
+excerpt: In 2030, governments only use open source - a thought experiment
+categories:
+  - News
+---
+
+# Exploring a world of open source
+
+At the Mozilla Virtual Festival last week, we organized a session: "In 2030, governments only use open source: a thought experiment". We wanted to explore a future where all public organizations in the world have joined us in our mission. The full recording is available on the [session page](https://schedule.mozillafestival.org/session/BRR3GV-1); this blog post is a summary of the discussion.
+
+One of the most significant changes would be the impact on public organizations that interact most with regular citizens, such as social services, county councils, public water, and electricity providers. They would have the most practical reasons to switch to open source. This would also apply to smaller outfits with limited funding as they could capitalize on shared codebases. More open source would also provide more modern, user-friendly digital services and solve more challenging problems because the basics would be solved collectively. Additionally, digital services would be more easily customized than white label proprietary software, meeting citizens' needs better.
+
+Another area that would see improvements would be government administration. It could promote transparency and accountability, which could lead to greater trust from citizens. Although making the code public would reveal inequality and conflicts of interest, this does not mean everyone would understand it. Therefore, it may require translating the code into lay terms. Following that, it would be possible that non-governmental organizations (NGOs) and public safety sectors could also benefit from new business models.
+
+One potential issue with the adoption of open source is the risk of less diversity among codebases which would risk a homogenization in the organizations using them. One participant reflected that it would be a kind of colonialism through software. This could be mitigated through advanced configuration options or even local distributions or forks to make it possible to customize it properly, but to do that, it is important to see whether there is a need and be open to suggestions from organizations that want to use the codebase.
```

Comment:
```suggestion
One potential issue with the complete adoption of open source is the risk of less diversity among codebases, which would risk a homogenization in the organizations using them. One participant reflected that it would be a kind of colonialism through software. This could be mitigated through advanced configuration options or even local distributions or forks to make it possible to customize it properly, but to do that, it is important to see whether there is a need and be open to suggestions from organizations that want to use the codebase.
```